protocol/igmp.go

```diff
@@ -188,15 +188,15 @@ func (p *IGMPv3Query) UnmarshalBinary(data []byte) error {
 	p.Checksum = binary.BigEndian.Uint16(data[n:])
 	n += 2
 	p.GroupAddress = data[n : n+4]
-	n += 5
+	n += 4
```

Comment:
are the changes in this file bug fixes? If yes, could you mention it in the commit message?

Reply:
Yes, added in the commit message.